tools/windows-node-installer/test/e2e/utils.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/masterzen/winrm"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/types"
+	"strconv"
+	"time"
+)
+
+const (
+	// winRMPort is the secure WinRM port
+	winRMPort = "5986"
+)
+
+var (
+	// stores windows instance credential information.
+	credentials *types.Credentials
+	user        = "Administrator"
+)
+
+// setupWinRMClient initializes the winRM client and executes commands on the windows node
+func setupWinRMClient(command string) error {
```

Comment:
This is a misnomer, this function is not just setting up the winrm client, it is executing a winrm command with setup being a required step.

Please either rename this function accordingly or preferably, remove the part that executes a winrm command from the function

Reply:
I agree with @sebsoto. You need to separate the two concerns here. Have a function that setups the winrm client and call it from the setup function in both aws and azure. You can then have a generic run function. That returns stdout and stderr. Please see the `WindowsVM` code that I added.